app/controllers/propertyLinking/DeclarationController.scala

```diff
@@ -72,18 +72,17 @@ class DeclarationController @Inject()(
       .map { _ =>
         val evidenceType: Option[EvidenceType] = request.ses.uploadEvidenceData.fileInfo.map(_.evidenceType)
         val capacityType: Option[CapacityType] = request.ses.propertyRelationship.map(_.capacity)
-
         val evidenceChoice: Option[EvidenceChoices.Value] = evidenceType.zip(capacityType).headOption.map {
-          case (RatesBillType, Owner | OwnerOccupier)        => EvidenceChoices.RATES_BILL
-          case (ServiceCharge, Owner | OwnerOccupier)        => EvidenceChoices.SERVICE_CHARGE
-          case (StampDutyLandTaxForm, Owner | OwnerOccupier) => EvidenceChoices.STAMP_DUTY
-          case (LandRegistryTitle, Owner | OwnerOccupier)    => EvidenceChoices.LAND_REGISTRY
-          case (WaterRateDemand, Owner | OwnerOccupier)      => EvidenceChoices.WATER_RATE
-          case (OtherUtilityBill, Owner | OwnerOccupier)     => EvidenceChoices.UTILITY_RATE
-          case (Lease, Occupier | Owner | OwnerOccupier)     => EvidenceChoices.LEASE
-          case (License, Occupier | Owner | OwnerOccupier)   => EvidenceChoices.LICENSE
-          case (_, Owner | OwnerOccupier)                    => EvidenceChoices.OTHER
-          case (_, Occupier)                                 => EvidenceChoices.NO_LEASE_OR_LICENSE
+          case (RatesBillType, Owner | OwnerOccupier | Occupier)        => EvidenceChoices.RATES_BILL
+          case (ServiceCharge, Owner | OwnerOccupier | Occupier)        => EvidenceChoices.SERVICE_CHARGE
+          case (StampDutyLandTaxForm, Owner | OwnerOccupier | Occupier) => EvidenceChoices.STAMP_DUTY
+          case (LandRegistryTitle, Owner | OwnerOccupier | Occupier)    => EvidenceChoices.LAND_REGISTRY
+          case (WaterRateDemand, Owner | OwnerOccupier | Occupier)      => EvidenceChoices.WATER_RATE
+          case (OtherUtilityBill, Owner | OwnerOccupier | Occupier)     => EvidenceChoices.UTILITY_RATE
+          case (Lease, Occupier | Owner | OwnerOccupier | Occupier)     => EvidenceChoices.LEASE
+          case (License, Occupier | Owner | OwnerOccupier | Occupier)   => EvidenceChoices.LICENSE
+          case (_, Owner | OwnerOccupier | Occupier)                    => EvidenceChoices.OTHER
+          case (_, Occupier)                                            => EvidenceChoices.NO_LEASE_OR_LICENSE
```

Comment:
this case (_, Occupier) will never happen anymore right? now that youve added the Occupier to the above case statement, so its not needed?